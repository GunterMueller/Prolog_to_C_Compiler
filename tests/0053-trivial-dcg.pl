main :-
	rule("yo!", X), display(X), nl.

rule, "done" --> "yo".

