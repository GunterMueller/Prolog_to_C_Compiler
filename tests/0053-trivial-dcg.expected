[100,111,110,101,33]
